services/device-registry-base/src/main/java/org/eclipse/hono/deviceregistry/service/tenant/AutowiredTenantInformationService.java

```diff
@@ -29,17 +28,18 @@
 import io.vertx.core.Future;
 
 /**
- * A default implementation of {@link TenantInformationService} that uses embedded {@link TenantService} to verify if tenant exists.
+ * A default implementation of {@link TenantInformationService} that uses embedded {@link TenantManagementService} to
```

Comment:
How about leaving this class as it is but instead create a new `TenantServiceImplementation` that uses `TenantManagementService` and does not rely on Spring auto-wiring anymore?

Reply:
Ok, but still anyway the `AutoWiredTenantInformationService` need to updated with `TenantManagementService` as we use now `Tenant` instead of `TenantObject`.

Reply:
obviously